fast64_internal/sm64/sm64_geolayout_writer.py

```diff
@@ -433,22 +428,49 @@ def convertArmatureToGeolayout(armatureObj, obj, convertTransformMatrix, camera,
             None,
             None,
             None,
-            meshGeolayout.nodes[i],
+            i,
             [],
             name,
             meshGeolayout,
             geolayoutGraph,
             infoDict,
             convertTextureData,
         )
+
+    def walk(node, fMeshes):
+        base_node = node.node
+        if type(base_node) == JumpNode:
+            if base_node.geolayout:
```

Comment:
I think it'd be better with `if base_node.geolayout is not None` to make the intent clearer